disco_aws_automation/disco_metanetwork.py

```diff
@@ -63,27 +62,21 @@ def _tag_resource(self, resource, suffix=None):
         keep_trying(300, resource.add_tag, "meta_network", self.name)
 
     @property
```

Comment:
I was trying to avoid querying AWS every time this method is called when `self._centralized_route_table` is `None`. I can add a member variable to indicate the query has been made once and not to be made again.